src/display/api.js

```diff
@@ -951,17 +951,60 @@ var PDFPageProxy = (function PDFPageProxyClosure() {
     },
 
     /**
+     * @param {boolean} normalizeWhitespace - replaces all occurrences of
+     * whitespace with standard spaces (0x20). The default value is `false`.
+     * @param {boolean} disableCombineTextItems - do not attempt to combine
+     * same line {@link TextItem}'s. The default value is `false`.
+     * @return {ReadableStream} ReadableStream to read textContent chunks.
+     */
+    streamTextContent({ normalizeWhitespace = false,
+                        disableCombineTextItems = false, }) {
```

Comment:
Currently, it's optional to provide an options object to `getTextContent` and it'd be a really good idea if the new method has a similar signature!
I think that the following should work to make the object optional (otherwise just use the `params` notation as below):
```javascript
streamTextContent({ normalizeWhitespace, disableCombineTextItems, } = {}) {
```